new-lamassu-admin/src/pages/OperatorInfo/OperatorInfo.styles.js

```diff
@@ -24,16 +24,32 @@ const styles = {
   },
   row: {
     display: 'flex',
+    alignItems: 'center',
     justifyContent: 'space-between',
     marginBottom: 28,
     width: 600,
     '&:last-child': {
       marginBottom: 0
     }
   },
+  switchRow: {
+    display: 'flex',
+    alignItems: 'center',
+    marginBottom: 28,
+    width: 600,
+    '&:last-child': {
+      marginBottom: 0
+    }
```

Comment:
does last child change? can't we just put this on another class?

Reply:
> does last child change? can't we just put this on another class?

I checked and It actually don't have any practical effect as of now, so I just removed it.